### ISSUE TITLE
Xcode 13.1 "invalid exclude" warning

### ISSUE THREAD

**Issue by bbauman1** [realm/realm-swift#7624]
### How frequently does the bug occur?

All the time

### Description

When compiling with Realm through SPM a warning is generated for an Invalid Exclude

### Stacktrace & log output

```shell
Invalid Exclude '/Users/redacted/Library/Developer/Xcode/DerivedData/redacted/SourcePackages/checkouts/realm-cocoa/tools': File not found.
```

### Can you reproduce the bug?

Yes, always

### Version

10.21.1

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

Xcode 13.1. iOS any-version

### Build environment

Xcode version: 13.1
Dependency manager and version: SPM

**Comment by bbauman1** [comment]
I was able to get a more descriptive output 

```Invalid Exclude '/Users/redacted/Library/Developer/Xcode/DerivedData/redacted/SourcePackages/checkouts/realm-core/src/realm/object-store/impl/set_notifier.hpp': File not found.```

**Comment by leemaguire** [comment]
Hi @bbauman1 We will fix this, thanks!

**Comment by drmarkpowell** [comment]
seeing this in Xcode 13.2.1 as well

**Comment by zbeckman** [comment]
Same issue. Xcode 13.2.1, Realm 10.21.1 (and RealmDatabase 11.8.0).
> Invalid Exclude '/Users/zbeckman/Library/Developer/Xcode/DerivedData/NourishMe-gfsnyvfufdqacwdlkrekaaxqhgsm/SourcePackages/checkouts/realm-swift/tools': File not found.

**Comment by aehlke** [comment]
Does latest Xcode work with 10.21.0 or an earlier version instead?

**Comment by OlegEremenko991** [comment]
Same issue:
Invalid Exclude '/Users/oleg991/Library/Developer/Xcode/DerivedData/Days_2.1-cuqmlgwmfaqgzgdzjktdrpjgkshc/SourcePackages/checkouts/realm-cocoa/tools': File not found.

**Comment by pavel-ship-it** [comment]
Fixed in #7646
